datahub-graphql-core/src/main/resources/entity.graphql

```diff
@@ -582,6 +582,15 @@ type Mutation {
     """
     createPost(input: CreatePostInput!): Boolean
 
+    """
+    Update a post
+    """
+    updatePost(
+        "The urn of the View to update"
+        urn: String!,
```

Comment:
Please move `urn` inside `UpdatePostInput`  type and make changes on java side accordingly. This is for consistency with other graphql endpoint Inputs

Reply:
Sure, I will move it though I followed the existing _UpdateViewInput_ type.